flipperdevices/flipperzero-firmware
# [FL-2847] FFF trailing space fix

# What's new

- Improve FFF robustness against stray whitespace characters

# Verification 

1. Compile the firmware with unit tests enabled and flash.
2.  Run `unit_tests flipper_format`. The test should pass.
3. Compile the firmware normally and flash it.
4. Download [this file](https://github.com/flipperdevices/flipperzero-firmware/files/9686275/Test.ir.zip), unzip it (GitHub isn't allowing to upload it direcly) and put it into the `/ext/infrared` folder.
5. Go to `Infrared -> Saved Remotes -> Test`. Two entries `Signal_1` and `Signal_2` should be listed.
6. (Optional) Compile and flash the `dev` firmware branch.
7. (Optional) Repeat step 5. No entries should be listed.

# Checklist (For Reviewer)
